dev-haskell/patch/patch-0.0.8.4.ebuild

```diff
@@ -15,26 +15,27 @@ HOMEPAGE="https://obsidian.systems"
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
+IUSE="+test"
```

Comment:
I'm not sure this should be force-enabled, as that will force-enable the test-suite which is unnecessary.